packages/forklift-console-plugin/src/modules/Providers/views/create/components/VSphereProviderCreateForm.tsx

```diff
@@ -51,7 +51,10 @@ export const VSphereProviderCreateForm: React.FC<VSphereProviderCreateFormProps>
     validation: {
       url: {
         type: 'default',
-        msg: 'The URL of the vSphere API endpoint for example: https://host-example.com/sdk .',
+        msg:
```

Comment:
the end goal is to make separate components for eacy type `show A`  and `show B` in places we now have components that do `if type A show A and if type B show B`

this PR adds a new `if ...` where the direction should be to split the component to the two different types